Makefile

```diff
@@ -2,6 +2,7 @@ PYTHON		?= python3
 
 PKGNAME=rhc-worker-playbook
 PKGVER = $(shell $(PYTHON) setup.py --version | tr -d '\n')
+RELEASE = $(shell printf "99.%s.git.%s" $(shell git rev-list $(shell git describe --tags --abbrev=0 --always | tr -d '\n')..HEAD --count | tr -d '\n') $(shell git rev-parse --short HEAD | tr -d '\n'))
```

Comment:
Can you please split this line using `\` into several lines to improve readability?

Reply:
Would you prefer to break all the individual subshell "$(shell)" calls into
named variables? Maybe that would improve readability more than just some newlines.

Reply:
Your change is OK. It is easy to read it now. :+1: